keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -63,8 +54,24 @@
     ]
 )
 
+# Matches CJK characters.
```

Comment:
Let's add a link to the place we got this from in the BERT repo. It's a good reference to track.